runtime/src/iree/hal/drivers/hip/stream_command_buffer.c

```diff
@@ -501,9 +473,6 @@ static iree_status_t iree_hal_hip_stream_command_buffer_dispatch(
   iree_hal_hip_stream_command_buffer_t* command_buffer =
       iree_hal_hip_stream_command_buffer_cast(base_command_buffer);
   IREE_TRACE_ZONE_BEGIN(z0);
```

Comment:
nice refactoring of these out of here given that the stream command buffer is only ever executed on the worker thread!